components/navbar/ShoppingCartButton.vue

```diff
@@ -11,7 +11,7 @@
         v-if="numberOfItems"
         :count="numberOfItems"
         rounded
-        class="count-position text-xs" />
+        class="right-[-0.5rem] top-[-0.5rem] !left-[unset] !bottom-[unset] text-xs" />
```

Comment:
```suggestion
        position="top-right"
        class="!right-[-0.5rem] top-[-0.5rem] text-xs" />
```

Reply:
It's not correct because you have to override `bottom` and `right` if you stick to do so.

<img width="242" alt="image" src="https://github.com/kodadot/nft-gallery/assets/31397967/c2f279e7-3031-4d58-902c-49a5a47ede99">

Reply:
my bad, still look weird to have custom properties like this
![Screenshot 2024-01-19 at 09-52-15 KodaDot - NFT Market Explorer](https://github.com/kodadot/nft-gallery/assets/9987732/88a28d2c-4272-4319-8f27-2bfc6b03d9b1)
![Screenshot 2024-01-19 at 09-53-00 KodaDot - NFT Market Explorer](https://github.com/kodadot/nft-gallery/assets/9987732/375736e2-934e-49a3-a2ab-723ea5019608)